metadata/latest/Azure.AI.FormRecognizer.json

```diff
@@ -1,6 +1,17 @@
 {
   "Name": "Azure.AI.FormRecognizer",
   "Version": "4.0.0",
+  "sdkCSV": {
```

Comment:
Why have this nested instead of having the properties directly on the object?

Reply:
I am thinking to loop into sdkCSV everytime we update in docindex. So we don't need to worry about which field takes precedence, like the field `isNewSDK`, `type`, these are duplicated fields.

Reply:
Doesn't that just move the work that needs to figure out what takes precedence? We would need to interpret that at some point.

Reply:
Two situations here:
1. Release pipeline will update docs metadata most up-to-date. So in this case, I'd take the value from docs metadata.
2. For the package out of release pipeline, I'd take the value from csv files. 

If we leave one for each field only, then in docindex pipeline we need to figure out whether we override or not. I don't think it is easy to decide.
For example, Microsoft.Azure.Storage.Blobs are outside of release pipeline, so it will look like
```
{
 Name: Microsoft.Azure.Storage.Blobs,
 Version: 1.1.1,
 Type: client,
 New: true;
}
```
In docindex, we are lack of ability to determine whether to update `type` and `new` from csv file. (docindex can't tell if it is from release pipeline or not)

If we use sdkCSV, then we can blindly update the block as the json will look like:
```
{
 Name: Microsoft.Azure.Storage.Blobs,
 Version: 1.1.1,
 sdkCSV: {
  Type: client;
  New: true;
 }
}
```

When we need to use type and new info for filter or toc updates, we can take the out layer block value first, if no such fields, then take inner block value.